content/docs/user-guide/interactivity/robotics/importing-robot/sdformat-plugins.md

```diff
@@ -31,6 +31,22 @@ _ROS2AckermannModel_ is a pre-defined hook used to map `libgazebo_ros_ackermann_
 
+#### O3DE Joint State Publisher
+
+_ROS2JointStatePublisherModel_ is a pre-defined _hook_ used to map `libgazebo_ros_joint_state_publisher.so` SDFormat plugin into the _JointsManipulationEditorComponent_ in O3DE. The component is added to the base link of the robot alongside either _JointsArticulationControllerComponent_ or _JointsPIDControllerComponent_ depending on whether you decide to use articulations in your imported robot.
+
+Note that, as the namespace policy in O3DE differs from Gazebo, this hook won't change the namespace on which joint states are published whether or not you specified it in the SDFormat file. Similarly, you won't be able to specify the names of the joints to be published; O3DE _JointsManipulationEditorComponent_ will publish the states of each joint either way.
+
+#### O3DE Joint Pose Trajectory
+
+_ROS2JointPoseTrajectoryModel_ is a pre-defined _hook_ used to map `libgazebo_ros_joint_pose_trajectory.so` SDFormat plugin into the _JointsTrajectoryComponent_ in O3DE. The component is added to the base link of the robot alongside _JointsManipulationEditorComponent_ and either _JointsArticulationControllerComponent_ or _JointsPIDControllerComponent_ depending on whether you decide to use articulations in your imported robot. Similarly to Joint State Publisher hook, namespace remapping won't be parsed here.
+
+If you plan on importing both Joint State Publisher and Joint Pose Trajectory in one SDFormat file, Joint State Publisher might not get imported because of the order in which plugins are listed in your file. If that happens, _JointsManipulationEditorComponent_ will be added anyway, but parameters of created component might differ from those specified in your robot description. Information about it will be visible in the Robot Importer summary window. For now, there are two ways to resolve this issue:
+1. Make sure that `libgazebo_ros_joint_state_publisher.so` plugin is listed before `libgazebo_ros_joint_pose_trajectory.so` in your SDFormat file. If not, you can change their order manually by simply swapping their descriptions (including headlines as well as plugin parameters) using copy-paste.
+2. After performing succesful import of your SDFormat file (excluding `libgazebo_ros_joint_state_publisher.so`), go to the base link of your model and manually adjust parameters of the _JointsManipulationEditorComponent_. Note that since this O3DE component serves not only for publishing joint states, but also for actions like setting joints start positions, you might encounter more parameters avilable than just those that define the way your joint states will be published. The parameters that you might need to adjust are all visible under the _Joint State Publisher_ section.
+
+If you decide to use articulations in your imported robot, you might need to enable motor in articulation links of your joints manually in order to be able to adjust their trajectory.
```

Comment:
I believe the same adjustment is required for joints, isn't it?

Btw. we should push the changes to O3DE at some point to enable changing this setting.